Makefile

```diff
@@ -1,3 +1,26 @@
+COMMIT_HASH := $(shell git rev-parse HEAD)
+# ORB_VERSION defaults to a dev version based on the git hash, but can be
+# overridden with a semver to publish a release.
```

Comment:
So circle magically knows if the tag is semver it's a "release"? sketchy

Reply:
The "magic" is a regex in the circle config, also documented in the README. Open to alternatives though